src/App.js

```diff
@@ -1,5 +1,25 @@
+import { parsePlayers, parseRound } from './parseInput';
+import { processInputToOutput } from './Simulate';
+import { formatOutput } from './formatOutput';
+
+import { MissionUtils } from '@woowacourse/mission-utils';
+const Console = MissionUtils.Console;
```

Comment:
```suggestion
import { Console } from '@woowacourse/mission-utils';
```
이렇게 바로 `Console` 객체를 import 하는건 어떨까요??

Reply:
내부 객체를 바로 import할 수 있군요! 몰랐는데 알려주셔서 감사합니다!